src/sandstorm/supervisor.capnp

```diff
@@ -274,15 +256,28 @@ struct ApiTokenOwner {
 
       introducerIdentity @9 :Text;
       # The identity ID through which a user's powerbox action caused the grain to receive this
-      # token. This is the identity against which the `requiredPermissions` parameter
-      # to `restore()` will be checked. This field is only intended to be filled in by the
-      # front-end during a powerbox request; a regular `save()` call produces a capability that
-      # has no "introducer".
+      # token. The token will become invalid if this identity loses access to the grain. This field
```

Comment:
Is there a good reason not to declare `grain.introducerIdentity` entirely obsolete and ignore it?